py_ballisticcalc/settings.py

```diff
@@ -1,34 +1,39 @@
 """Global settings of the py_ballisticcalc library"""
 import logging
-
+import dataclasses
 from .unit import Unit, Distance
 
 __all__ = ('Settings',)
 
+class Metadataclass(type):
+    """Provide representation method for static dataclasses."""
+    def __repr__(cls):
+        return '\n'.join(f'{field.name} = {getattr(cls, field.name)!r}'
+                    for field in dataclasses.fields(cls))
 
 class Settings:  # pylint: disable=too-few-public-methods
     """Global settings class of the py_ballisticcalc library"""
 
-    class Units:  # pylint: disable=too-few-public-methods
+    @dataclasses.dataclass
+    class Units(metaclass=Metadataclass):  # pylint: disable=too-many-instance-attributes
         """Default units for specified measures"""
-
-        sight_height: Unit = Unit.INCH
-        twist: Unit = Unit.INCH
-        velocity: Unit = Unit.FPS
+        angular: Unit = Unit.DEGREE
         distance: Unit = Unit.YARD
-        temperature: Unit = Unit.CELSIUS
-        weight: Unit = Unit.GRAIN
-        length: Unit = Unit.INCH
+        velocity: Unit = Unit.FPS
+        pressure: Unit = Unit.IN_HG
+        temperature: Unit = Unit.FAHRENHEIT
         diameter: Unit = Unit.INCH
-        pressure: Unit = Unit.HP
-        drop: Unit = Unit.CENTIMETER
-        angular: Unit = Unit.DEGREE
+        length: Unit = Unit.INCH
+        weight: Unit = Unit.GRAIN
         adjustment: Unit = Unit.MIL
-        energy: Unit = Unit.JOULE
+        drop: Unit = Unit.INCH
+        energy: Unit = Unit.FOOT_POUND
         ogw: Unit = Unit.POUND
+        sight_height: Unit = Unit.INCH
         target_height: Unit = Unit.INCH
+        twist: Unit = Unit.INCH
 
-    _MAX_CALC_STEP_SIZE: float = 1
+    _MAX_CALC_STEP_SIZE: float = 0.5
```

Comment:
when and how we can remove this redundant property?

Reply:
I don't understand: How is this redundant?  (In any case, this may get modified when I finish [this issue](https://github.com/dbookstaber/py_ballistics/issues/2).)

Reply:
Yes I'm about turning to time-step calculation